effectivity/README.md

```diff
@@ -0,0 +1,158 @@
+--- 
```

Comment:
Please separate the rest of the patterns into their own pull requests. This is too big to review all at once.

Reply:
No worries, Ill link them in the comments once I have the requests ready.